python/pyabacus/src/pyabacus/ModuleBase/_module_base.py

```diff
@@ -0,0 +1,105 @@
+"""
+pyabacus.ModuleBase
+===================
+Basic math functions and integrals.
+"""
+
+import numpy as np
+
+from typing import overload
+from numpy.typing import NDArray
+
+class Sphbes:
+    def __init__(self) -> None: ...
+    @overload
+    @staticmethod
+    def sphbesj(l: int, x: float) -> float: ...
+    @overload
+    @staticmethod
+    def sphbesj(
+        n: int, 
+        r: NDArray[np.float64], 
+        q: int, 
+        l: int, 
+        jl: NDArray[np.float64]
+    ) -> None: ...
+    @overload
+    @staticmethod
+    def dsphbesj(l: int, x: float) -> float: ...
+    @overload
+    @staticmethod
+    def dsphbesj(
+        n: int, 
+        r: NDArray[np.float64], 
```

Comment:
I was wondering whether it was necessary to add function signatures in python's `__init__.py`. Plz refer to this repo's [example](https://gitlab.com/veloxchem/veloxchem/-/blob/master/src/pymodule/__init__.py?ref_type=heads), maybe the suitable code looks like
` from ._core import ModuleBase`

Reply:
hmm...Using only Pybind, we cannot enable IDEs to provide type hints and code completion. Adding function signatures in Python files can achieve this.

Reply:
I see. You can keep them in your local changes (since they only provide convenience for coding not for usage), but keep `__init__` with only necessary information like the example I show you.